grafana-plugin/src/containers/AlertRules/AlertRules.tsx

```diff
@@ -63,6 +62,14 @@ interface AlertRulesState {
   editIntegrationName?: string;
 }
 
+const Notification: React.FC = () => (
```

Comment:
this JSX was duplicated twice at the bottom of this file, so I extracted it out into its own component